### PR TITLE
Updated deploy-hybrid.md

### PR DESCRIPTION
**Pull request by dariomws**
Closes https://github.com/MicrosoftDocs/OfficeDocs-Exchange/issues/2195.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeHybrid/hybrid-deployment/deploy-hybrid.md [hunk 1]
```diff
@@ -95,6 +97,9 @@ Use the following procedure to create and configure a hybrid deployment:
 
 11. On the **Federation Trust** page, click **Enable** and then click **Next**.
 
+    > [!NOTE]
+    > Step 11 and 12 will appear only if there are Exchange 2010 servers on-premises.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeHybrid/hybrid-deployment/deploy-hybrid.md:101]:
```suggestion
    > Steps 11 and 12 will appear only if there are Exchange 2010 servers on-premises.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edit.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@msdmaguire PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**